components/internal/modals/EditPermitHolderInformationModal.tsx

```diff
@@ -0,0 +1,186 @@
+import {
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalFooter,
+  ModalBody,
+  Button,
+  FormControl,
+  FormLabel,
+  Input,
+  useDisclosure,
+  Text,
+  Stack,
+  FormHelperText,
+  Box,
+  Divider,
+} from '@chakra-ui/react'; // Chakra UI
+import { useState, SyntheticEvent } from 'react'; // React
+
+export default function EditPermitHolderInformationModal() {
+  const { isOpen, onClose, onOpen } = useDisclosure();
+
+  // Personal information state
+  const [firstName, setFirstName] = useState('');
+  const [lastName, setLastName] = useState('');
+
+  // Contact information state
+  const [email, setEmail] = useState('');
+  const [phoneNumber, setPhoneNumber] = useState('');
+
+  // Home address information state
+  const [addressLine1, setAddressLine1] = useState('');
+  const [addressLine2, setAddressLine2] = useState('');
+  const [city, setCity] = useState('');
+  const [postalCode, setPostalCode] = useState('');
+
+  //   TODO: Add error states for each field (post-mvp)
+
+  const handleSubmit = (event: SyntheticEvent) => {
+    event.preventDefault();
+    // TODO: Will be addressed in API hookup
+  };
+
+  return (
+    <>
+      {/* Button will be removed before merging */}
+      <Button mt={3} onClick={onOpen}>
+        Open
+      </Button>
+
+      <Modal onClose={onClose} isOpen={isOpen} scrollBehavior="inside" size="3xl">
+        <ModalOverlay />
+        <form onSubmit={handleSubmit}>
+          <ModalContent paddingX="36px">
+            <ModalHeader
+              textStyle="display-medium-bold"
+              paddingBottom="12px"
+              paddingTop="24px"
+              paddingX="4px"
+            >
+              <Text textStyle="display-medium-bold">{'Edit Permit Holder Information'}</Text>
```

Comment:
Most of the `Text` components are missing `as` props, if you could go though and add it to each of them that would be great!

Reply:
Done!!